src/pages/account/logins/[type]/[id].tsx

```diff
@@ -53,7 +53,7 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
         return;
       }
 
-      const userSession = request.session.get<UserCookie>("user-session");
+      const userSession = request.session.get<UserCookie>("user-cookie");
```

Comment:
Should we let `userSession` here, since we changed all the rest to be named with "cookie" ?

Reply:
Nice catch 💪.

Reply:
83071fc
I searched through the codebase to change all of them.